Rakefile

```diff
@@ -5,9 +5,4 @@ require 'robot-controller/tasks'
 
 Dir.glob('lib/tasks/*.rake').each { |r| import r }
 
-require 'rspec/core/rake_task'
-
-desc 'Run specs'
-RSpec::Core::RakeTask.new(:spec)
-
-task default: [:yard, :spec]
+task default: [:yard, :spec, :rubocop]
```

Comment:
Would travis fail quicker if this array were `[:spec, :yard, :rubocop]` when the specs contain a failure?